server/app/views/applicant/ModalFragment.html

```diff
@@ -51,24 +51,6 @@ <h2 class="usa-modal__heading" id="debug-content-modal-heading">
   </dialog>
 </div>
 
-<div
-  th:fragment="programIndexLogin"
-  id="modal-container"
-  class="display-none position-fixed height-viewport width-full z-100"
->
-  <div th:each="programIdAndLoginBypassUrl: ${programIdsToLoginBypassUrls}">
```

Comment:
**suggestion (blocking):** As far as I can tell, this is the only place where the `programIdsToLoginBypassUrls` variable was used.  I suggest that we also remove this and any associated logic: [this line](https://github.com/civiform/civiform/blob/1dad17c1551369a86184fff64a537c0e7c91ae43/server/app/views/applicant/NorthStarProgramIndexView.java#L145  )

Reply:
Thanks!